wpilibNewCommands/src/main/native/cpp/frc2/command/CommandScheduler.cpp

```diff
@@ -332,11 +311,10 @@ void CommandScheduler::Cancel(Command* command) {
     return;
   }
 
-  auto find = m_impl->scheduledCommands.find(command);
-  if (find == m_impl->scheduledCommands.end()) {
+  if (!m_impl->scheduledCommands.contains(command)) {
     return;
   }
-  m_impl->scheduledCommands.erase(find);
+  m_impl->scheduledCommands.erase(command);
```

Comment:
Is there a way to eliminate the double lookup here?

Reply:
use a vector instead of smallset, as mentioned above